src/services/users.js

```diff
@@ -22,7 +23,9 @@ const createNewUser = async (user) => {
 	// eslint-disable-next-line max-len
 	const { rows, error } = await db.query(
 		// eslint-disable-next-line max-len
```

Comment:
I see you've broken down the line so I guess the eslint disable won't be needed anymore